go.mod

```diff
@@ -30,7 +30,7 @@ require (
 	github.com/apoydence/eachers v0.0.0-20181020210610-23942921fe77 // indirect
 	github.com/armon/go-socks5 v0.0.0-20160902184237-e75332964ef5
 	github.com/aws/aws-lambda-go v1.13.3
-	github.com/aws/aws-sdk-go v1.19.48
+	github.com/aws/aws-sdk-go v1.38.60
```

Comment:
@kaiyan-sheng As a follow up, would be good to figure out why we have 2 aws sdk ..